modules/_unstable/README.md

```diff
@@ -0,0 +1,44 @@
+# Canvas Kit Unstable
+
+This module is a sandbox for work in progress components.
+
+Since we have a rather rigorous process for getting components in at a production level, it can be
+valuable to make them available earlier for teams who need them during development. Many of our
+components are contributions from internal product teams. While these components can suit the
+contributing product's use case, it takes time to iterate on the API/functionality as we evaluate
+them in a more global context. The Unstable module allows us to do that as needed.
+
+## Breaking Changes
+
+Due to the nature of this bundle of components, we will be continuously iterating, likely making
+breaking changes. The goal of this module is to work unconstrained by our design system's semantic
+versioning. This means that **breaking changes can be deployed to
+`@workday/canvas-kit-react-unstable` or `@workday/canvas-kit-css-unstable` at any time and it will
+not trigger a major version bump across our other modules**. Since we use a fixed version number for
+all components, all changes/additions to this module will only result in a patch version bump across
```

Comment:
Are we confirmed that our version bumps will always be patch versions for updates to `unstable`?

Reply:
I guess we don't really have a way to restrict this. If someone uses `feat: ...` for a commit message it will automatically be a minor bump

Reply:
Will `unstable` have a different cadence than the rest of the modules? Do we keep all modules on the same version? (right now they are all `3.0.0-alpha.6`)

Reply:
Now that we've switched to fixed versioning with Lerna, I don't think there is a way to release it with a different version/cadence unless we explicitly ignore it in lerna.json. Is this something we want to do?

Reply:
🤷‍♂ I could imagine if `unstable` was on a different and possibly more frequent cadence that `unstable` could be using code changes from "stable" that hasn't been released yet and that causing issues.

I think it still makes sense to tie `unstable` to the same fixed releases as the rest of the packages. If internal teams want a faster cadence than our standard releases, we can create internal releases or public releases with a special tag.